deploy/backplane/cee/02-cee-machineset-ClusterRoleBinding.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: backplane-cee-machineset-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name:   name: backplane-cee-machineset-readers
```

Comment:
Duplicate "name" here.

Reply:
Yeah, I cleaned that and CI is still running